localstack/services/s3/s3_listener.py

```diff
@@ -129,7 +129,11 @@ def send_notifications(method, bucket_name, object_path):
             action = {'PUT': 'ObjectCreated', 'POST': 'ObjectCreated', 'DELETE': 'ObjectRemoved'}[method]
             # TODO: support more detailed methods, e.g., DeleteMarkerCreated
             # http://docs.aws.amazon.com/AmazonS3/latest/dev/NotificationHowTo.html
-            api_method = {'PUT': 'Put', 'POST': 'Post', 'DELETE': 'Delete'}[method]
+            if action == 'ObjectCreated' and method == 'POST':
+                api_method = 'CompleteMultipartUpload'
```

Comment:
Could you explain why this is needed? I did some testing with a similar patch that did not include this change.

Reply:
I don't know that it is needed, but when testing against AWS S3 SQS notifications, and then moving over to Localstack, I noticed that the `eventName` coming back from AWS and Localstack were different, so I tried to make them match for this case.